### ISSUE TITLE
Wrong relationship

### ISSUE THREAD

**Issue by poweruser82** [jeena/Bungloo#242]
I just followed a new entity. The profile view says `you both follow each other` but that entity doesn't follow me.

![schermata 2013-07-17 alle 13 30 35](https://f.cloud.github.com/assets/325193/811449/265b61a6-eed5-11e2-8b5f-6a7917c86eb5.png)

**Comment by jeena** [comment]
Yes the code is buggy but the thing is that I have to rewrite this part from scratch anyway for 0.3 so after that it will hopefully be fixed. I have to add that this functionality will not be in the first versions of Bungloo for 0.3, it will be added later on.

**Comment by poweruser82** [comment]
I know. It was just to keep track of the issue. (and make the counter higher :-P )
